components/button-cards/solution-card/solution-card.scss

```diff
@@ -0,0 +1,136 @@
+.card-row > div {
```

Comment:
I think this css already exists with generic-cards so we don't need to repeat this code in every card we want to use card-row, maybe a better approach should be to create a cards.scss file and setup some of the repeated code there instead